src/services/constants.service.ts

```diff
@@ -7,6 +7,7 @@ export class ConstantsService {
     static readonly disableFaviconKey: string = 'disableFavicon';
     static readonly disableAutoTotpCopyKey: string = 'disableAutoTotpCopy';
     static readonly enableAutoFillOnPageLoadKey: string = 'enableAutoFillOnPageLoad';
+    static readonly enableAutoTotpCopyOnAutoFill: string = 'enableAutoTotpCopyOnAutoFill';
```

Comment:
Should be called `enableAutoTotpCopyOnAutoFillKey` to be consistent with the naming convention

Reply:
Fixed! 👍